### DISCUSSION TITLE
Distributions and Maven Central Deploy

### DISCUSSION THREAD

**Discussion by SquidXTV** [jreleaser/jreleaser#1752] [Q&A] [answered]
I use jreleaser in my `build.gradle.kts` to deploy to GitHub Releases and Maven Central.

For the GitHub Releases I had to define `distributions` like this:
```kts
jreleaser {
    distributions {
        // ...
    }
}
```
but for Maven Central I specified a `stagingRepository`:
```kts
deploy {
    maven {
        mavenCentral {
            create("sonatype") {
                active = Active.ALWAYS
                url = "https://central.sonatype.com/api/v1/publisher"
                stagingRepository("build/staging-deploy")
            }
        }
    }
}
```

My question is does the Maven Central deploy also use the distributions + stagingRepository?
Or does it just use the stagingRepository?

I would get duplicate stuff if it uses both artifacts from distributions and stagingRepository and thats
why I want to make sure before.

Thanks

**Comment by aalmiray** [comment, marked as answer]
No, it does not. the `deploy` section is only concerned (at the moment) with artifacts configured via the `stagingRepository` setting.

Distributions OTOH may be uploaded, released, and/or packaged. See https://jreleaser.org/guide/latest/concepts/workflow.html for more information on each step.
